example/base.html

```diff
@@ -4,16 +4,12 @@
     <title>Nugget.js Graphing Library</title>
     <script src="/dependencies/d3.js"></script>
     <script src="js/require.min.js"></script>
-    <script src="js/example.js" type="text/javascript"></script>
-    <script src="js/nugget_usage.js" type="text/javascript"></script>
     <link type="text/css" href="/css/nugget.css" rel="stylesheet">
     <link type="text/css" href="example.css" rel="stylesheet">
 </head>
 <body>
     <h1>Nugget</h1>
-    <h4>Line Graph Example, Pure d3</h4>
-    <svg id="line_graph" width="900" height="500"></svg>
-    <h4>Line Graph Example, Pure Nugget</h4>
+    <h4>Line Graph Example</h4>
```

Comment:
I find it much easier to debug outside of the main app, so I've cleaned up this example so we can use it for testing `YYComparison` guides and resizing. The stuff that was removed from this file was all old d3 sandbox code.